integrationtest/prerelease/data/mail-v1.0.0-rc1.txt

```diff
@@ -0,0 +1,58 @@
+First line is skipped
+Subject: [MajorRelease] sandbox v1.0.0-rc1
+
+
+
+v1.0.0-rc1
```

Comment:
Oooohhh! This opens a can of worms... (well maybe is not that bad)

In terms of release notes, if the pre-release now takes the notes, the following rc will just get the bug fixed as in a patch release? I think it should, but then the final release should pull the full changelog too (probably the one of the first rc and ignore all the following bug fixes, as they should just fix stuff introduced in the rcs). But this is something that we might need to figure out.

We can take care of this in a future PR of course....